psycop/common/model_training_v2/trainer/data/data_filters/test_data_filters.py

```diff
@@ -37,3 +40,15 @@ def test_regional_filter():
     filtered_df = regional_filter.apply(dataloader).collect()
 
     assert filtered_df.shape == (3, 2)
+
+
+def test_id_filter():
+    split_sequence = pl.Series("dw_ek_borger", [1])
+    dataloader = MockDataLoaderForFilters()
```

Comment:
Why not just call this a df? instead of a dataloader? isn't the intention with lazy object that it should be treated as the object (a dataframe).

Reply:
Dataloaders are more flexible in that we can for instance use the `ParquetVerticalConcatenator` to load from multiple files (e.g. splits), or a horizontal concatenater to combine different feature sets.